### PR TITLE
[Rooms list] restore the Element tabs : Fav/Direct/Rooms

### PR DESCRIPTION
**Pull request by yostyle**
#500

### CODE REVIEW

**Comment by phlniji** [comment]
There is an issue when creating a new discussion (or joining an existing one by the same mechanism).

After creating/discussing/reading, when we try to come back to the previous screen (by clicking on the back button in the navigation bar), we are not anymore on the right tab (People), but on the latest (Conversations), with the title of the People tab
create_discussion
.

File: Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift [hunk 1]
```diff
@@ -71,8 +69,9 @@ public class RecentsListService: NSObject, RecentsListServiceProtocol {
         .home: [.invited, .favorited, .directHome, .conversationHome, .lowPriority, .serverNotice, .suggested],
         .favourites: [.favorited],
         .people: [.directPeople],
-        .rooms: [.conversationRooms, .suggested,
-            .invited, .favorited, .directHome, .directPeople, .conversationHome, .serverNotice]
+        .rooms: [.conversationRooms, .suggested]
```
Comment (giomfo) [inline Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift:72]: please add `.serverNotice` in this rooms list

**Review by giomfo — COMMENTED** [review]
Commented, tests ok except the case already observed by @phlniji

**Comment by giomfo** [comment]
@yostyle please add a change logs file

File: Tchap/Assets/Images.xcassets/Common/create_room.imageset/Contents.json [hunk 2]
```diff
@@ -5,16 +5,6 @@
       "idiom" : "universal",
```
[the remark this answers is not in the document]
Comment (yostyle) [reply Tchap/Assets/Images.xcassets/Common/create_room.imageset/Contents.json:5]: Probably but i'm not sure at this time.

File: Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift (hunk 1 above)
Comment (yostyle) [reply Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift]: Done

File: Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift [hunk 3]
```diff
@@ -71,8 +69,9 @@ public class RecentsListService: NSObject, RecentsListServiceProtocol {
...
+//        .rooms: [.conversationRooms, .suggested,
+//            .invited, .favorited, .directHome, .directPeople, .conversationHome, .serverNotice]
```
[the remark this answers is not in the document]
Comment (yostyle) [reply Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift]: Removed

**Review by giomfo — APPROVED** [review]
LGTM

**Pull request merged**